FirebasePerformance/Sources/FPRClient.m

```diff
@@ -107,6 +111,13 @@ - (instancetype)init {
     _eventsQueue = dispatch_queue_create("com.google.perf.FPREventsQueue", DISPATCH_QUEUE_SERIAL);
     _eventsQueueGroup = dispatch_group_create();
     _configuration = [FPRConfigurations sharedInstance];
+    NSString *plistPath = [[NSBundle mainBundle] pathForResource:@"GoogleService-Info"
+                                                          ofType:@"plist"];
+    NSMutableDictionary *googleServiceDict =
+        [NSMutableDictionary dictionaryWithContentsOfFile:plistPath];
+    _projectId = googleServiceDict[@"PROJECT_ID"];
+    _bundleId = googleServiceDict[@"BUNDLE_ID"];
```

Comment:
You can get the bundle ID from here https://github.com/firebase/firebase-ios-sdk/blob/master/FirebaseCore/Sources/Public/FirebaseCore/FIROptions.h#L43.